INSIGHTS_IMPROVEMENTS.md

```
@@ -0,0 +1,194 @@
+# 🚀 תוכנית שיפורים לצ'אט התבונות (Insights Chat)
+
+## 📋 סטטוס: בתהליך
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)